src/DonorDashboards/Repositories/Donations.php

```diff
@@ -211,6 +212,7 @@ protected function getReceiptInfo( $payment ) {
 		$receiptArr = [];
 
 		$sectionIndex = 0;
+		$lineBreaks = [ '<br />','<br>','<br/>' ];
```

Comment:
Actually, since you did a really good job by handling the line breaks with CSS, this part of the code is not necessary. I'll explain below.

Reply:
removed: https://github.com/impress-org/givewp/compare/959aaa5b55ed...2f9f4a276fad